docs/Administrator Guide/Setting Up Volumes.md

```diff
@@ -374,15 +369,10 @@ volumes, but using dispersed subvolumes instead of replicated ones.
     > - Make sure you start your volumes before you try to mount them or
     > else client operations after the mount will hang.
 
-    > - GlusterFS will fail to create a distributed dispersed volume if more than one brick of a disperse set is present on the same peer.
-
-    > 
-
-    >         # gluster volume create <volname> disperse 3 server1:/brick{1..6}
-    >         volume create: <volname>: failed: Multiple bricks of a replicate volume are present on the same server. This setup is not optimal.
-    >         Do you still want to continue creating the volume? (y/n)
+    > - GlusterFS will fail to create a dispersed volume if more than one brick of a disperse set is present on the same peer.
 
-    > Use the `force` option at the end of command if you want to create the volume in this case.
+    >        # gluster volume create <volname> disperse 3 server1:/brick{1..3} server2:/brick{1..3}
+    >        volume create: <volname>: failed: Multiple bricks of a disperse volume are present on the same server. This setup is not optimal. Bricks should be on different nodes to have best fault tolerant configuration. Use 'force' at the end of the command if you want to override this behavior.
 
```

Comment:
This could be confusing and user might ask question "Do we need to have as many node as the number of bricks in a disperse volume, in this case 6 nodes?"
Just like an example of bad setup, we should also give an example of good setup, specially considering distributed disperse volume where 2 bricks can be hosted on same node if they belong to different subvol.

Reply:
Done

Reply:
Addressed both the comments from ashish